src/helics/common/AsioContextManager.h

```diff
@@ -27,17 +28,6 @@ SPDX-License-Identifier: BSD-3-Clause
 #include <string>
 #include <utility>
 
-// The choice for noexcept isn't set correctly in asio::io_context (including asio.hpp instead
```

Comment:
The reason we can remove this is because we raised the minimum boost requirement?

Reply:
I think so -- as far as I can tell from the comments in https://github.com/GMLC-TDC/HELICS/pull/670 it was to get the Boost 1.58 build working.

Reply:
where were we testing that?

Reply:
One of the old Travis CI builds -- the old logs aren't available now, but from the commit success/fail statuses on the PR and the comment I think it's likely the builds with newer versions of Boost compiled.

I'm not seeing compile errors with our current CI jobs either, so our new minimum Boost version should be setting `BOOST_NOEXCEPT`.